components/onboarding_tasks/onboarding_video_modal/onboarding_video_modal.tsx

```diff
@@ -48,6 +48,7 @@ const OnBoardingVideoModal = ({onExited}: Props) => {
             <Modal.Body>
                 <iframe
                     src='//fast.wistia.net/embed/iframe/3t4dpc0k6b'
+                    // eslint-disable-next-line react/no-unknown-property
```

Comment:
Despite the fact that this is a valid attribute and it exists in the iframe attributes in react, eslint throws an error. No known solution for this and I didn't want to globally ignore in the configuration of eslint.